Xamarin.Essentials/Screenshot/Screenshot.android.cs

```diff
@@ -8,11 +8,14 @@ namespace Xamarin.Essentials
 {
     public static partial class Screenshot
     {
-        static bool PlatformCanCapture =>
-            Platform.WindowManager.DefaultDisplay?.Flags.HasFlag(DisplayFlags.Secure) == false;
+        static bool PlatformIsCaptureSupported =>
+            true;
 
         static Task<ScreenshotResult> PlatformCaptureAsync()
         {
+            if (Platform.WindowManager?.DefaultDisplay?.Flags.HasFlag(DisplayFlags.Secure) == true)
+                throw new FeatureNotSupportedException("Unable to take a screenshot of a secure window.");
```

Comment:
Would this make sense to be something like `UnauthorizedAccessException` instead?

Reply:
Oh yeah. That is much better.